host-requirements.txt

```diff
@@ -4,63 +4,75 @@
 # complexity to the installation and operation of armory, so we encourage the
```

Comment:
I thought this file had been obsoleted (though not deleted).  I can find no reference to it in current `develop` except some outdated docs.